### DISCUSSION TITLE
Release 0.2.0 shows default React Native Welcome screen

### DISCUSSION THREAD

**Discussion by medonja** [NUMde/compass-numapp-frontend#203] [Q&A] [answered]
I have just tried to run the release v0.2.0 on Android, and it seems to only show the React Welcome screen (see attached screenshot) - this came a bit as a surprise, is that a wanted behaviour?

![image](https://user-images.githubusercontent.com/9157689/178988694-36242a29-94cb-4571-9d96-2dfd23e1a8f2.png)

**Comment by mbastian93** [comment, marked as answer]
Are you sure you entered the setup command correctly with the reference to the template (`--template https://github.com/NUMde/compass-numapp-frontend/tree/v0.2.0`)?

Does your source code match the code in the [compass-numapp-template directory](https://github.com/NUMde/compass-numapp-frontend/tree/main/compass-numapp-template)?

**Reply by medonja** [reply]
You are right, I have no idea what went wrong, but it seems to be working now after giving it a second try.
